### DISCUSSION TITLE
Sort order for content items in admin dashboard

### DISCUSSION THREAD

**Discussion by NickDarvey** [OrchardCMS/OrchardCore#8879] [Q&A]
Is there any means of sorting content types by a particular (content part's) field in the admin dashboard? We display a content type ordered by a SortOrder field and we'd like it to be ordered the same way in the CMS admin dashboard for our editors.

**Comment by Skrypt** [comment]
I believe you can override these views in your admin theme. Though, interesting suggestion. These content types are ordered alphabetically because we never really thought they would need to be ordered differently.
